proc_scripts/CPMG_phasing.py

```diff
@@ -2,7 +2,6 @@
 from sympy import symbols
 from proc_scripts import *
 import math
-fl = figlist_var()
 def center_CPMG_echo(s, axis='t2',fl=None):
```

Comment:
I'm fixing a ROLLBACK ☹☹☹☹☹☹☹!

Reply:
so, I set this up as a saved comment!  To clarify what it means: 

I'm assuming that this change broke your code.  If it did, that's typically a problem with the rest of the code, not with the change that I made.  Most of the time, you should be able to change the rest of the code (I'm assuming, e.g., that here one of your functions is missing an `fl=None` keyword argument + etc.) around the change that I made.

If you really can't figure it out or really think my change was in error, we can discuss, but I can't right now think of any case where it would be good to just revert my change the way that you did here.

Reply:
what was rollbacked? I'm confused... this references the line fl = figlist_var()....

Reply:
in b948a1, I removed this line.  In a13f20, you added it back in.